apps/timetable/admin/ui/css/skin.css

```diff
@@ -209,6 +209,36 @@
     border-right-color: #DDD;
 }
 
+/* Event type */
+.gh-batch-edit-events-container .gh-event-type::before {
+    background-color: #555;
+    color: #FFF;
+}
+
+.gh-batch-edit-events-container .gh-event-type[data-first=""]::before {
+    display: none;
+}
+
+.gh-batch-edit-events-container .gh-event-type[data-type="Field Trip"]::before {
+    background-color: #7D9D5C;
+}
+
+.gh-batch-edit-events-container .gh-event-type[data-type="Lab"]::before {
+    background-color: #FF944D;
+}
+
+.gh-batch-edit-events-container .gh-event-type[data-type="Lecture"]::before {
+    background-color: #19A3FF;
+}
+
+.gh-batch-edit-events-container .gh-event-type[data-type="Paper"]::before {
+    background-color: #62326E;
+}
+
+.gh-batch-edit-events-container .gh-event-type[data-type="Seminar"]::before {
+    background-color: #92308A;
```

Comment:
Paper and Seminar have very similar colours, maybe we can make one of them red-ish?